lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -5,46 +5,62 @@
 from rest_framework.reverse import reverse
 
 
-class GradedTotalSerializer(serializers.Serializer):
-    earned = serializers.FloatField()
-    possible = serializers.FloatField()
+class CourseGradeSerializer(serializers.Serializer):
+    percent = serializers.FloatField()
+    is_passing = serializers.SerializerMethodField()
+
+    def get_is_passing(self, grade):
+        return grade.passed
+
+
+class GradingPolicySerializer(serializers.Serializer):
+    assignment_policies = serializers.SerializerMethodField()
+    grade_range = serializers.SerializerMethodField()
+
+    def get_assignment_policies(self, grading_policy):
+        return [{
+            'type': assignment_policy['type'],
+            'weight': assignment_policy['weight'],
+        } for assignment_policy in grading_policy['GRADER']]
+
+    def get_grade_range(self, grading_policy):
+        return grading_policy['GRADE_CUTOFFS']
 
 
 class SubsectionSerializer(serializers.Serializer):
+    assignment_type = serializers.SerializerMethodField()
     display_name = serializers.CharField()
-    due = serializers.DateTimeField()
-    format = serializers.CharField()
-    graded = serializers.BooleanField()
-    graded_total = GradedTotalSerializer()
-    # TODO: override serializer
+    total_points = serializers.SerializerMethodField()
+    is_graded_assignment = serializers.SerializerMethodField()
```

Comment:
Similar to above with just potentially using the default name of `graded`.

Otherwise, nit: `is_graded_assignment` -> `has_graded_assignment`. In my mind, the subsection itself isn't graded, but something within it is.